tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -1,4 +1,4 @@
-# pylint: disable=missing-function-docstring, missing-module-docstring, unidiomatic-typecheck/
+# pylint: disable=missing-function-docstring, redefined-builtin, missing-module-docstring, unidiomatic-typecheck/
```

Comment:
The remaining codacy error was about `float` not `int` or `bool` so I don't see the relevance for issue #733